### ISSUE TITLE
Main window remains minimize (not pop-up) in windows

### ISSUE THREAD

**Issue by m-zakeri** [texstudio-org/texstudio#3099]
Texstudio (4.5.2) will open in Windows taskbar but it is not maximized anyway.

**Comment by octaeder** [comment]
If possible please provide a screenshot of the situation.

**Comment by denisclaude** [comment]
I think I have the same problem. Today, I decided to upgrade to TexStudio 4.5.2. The previous version was working properly. But, now, after the installation the welcome screen opens but after that nothing... In the windows task bar the application appears as active but the main windows does not appear.

**Comment by octaeder** [comment]
We may try if txs starts with defaults. So end texstudio.exe and ensure with task manager that this process is not running. Go to %AppData%/texstudio and rename texstudio.ini to texstudio~.ini. Now try to start texstudio.exe.

**Comment by m-zakeri** [comment]
> I think I have the same problem. Today, I decided to upgrade to TexStudio 4.5.2. The previous version was working properly. But, now, after the installation the welcome screen opens but after that nothing... In the windows task bar the application appears as active but the main windows does not appear.

Exactly, I had the same issue, and after **downgrading** to version 4.5.1 it was fixed!

**Comment by denisclaude** [comment]
Thank you very much Octaeder. I followed the steps that you indicated (I renamed the textstudio.ini file) and it fixed the problem.
Thank you so much for your help !

**Comment by sunderme** [comment]
never seen
